### ISSUE TITLE
[Enhancement] Factory make autocompletion

### ISSUE THREAD

**Issue by gtjamesa** [laravel-idea/plugin#102]
The autocompletion on the `FactoryBuilder::create()` method works as intended, however, if I use the `make()` method, I only receive autocompletion for the Eloquent methods and not those associated with the model:

![](https://i.imgur.com/GMqz07i.png)
![](https://i.imgur.com/I4XbHYE.png)

**Comment by adelf** [comment]
Hi, James.

Stupid mistake and nobody caught it before you :) The code was something like that:
```
            if (method.name == "create") {
                return "<ModelType>"
            }

            if (method.name == "createMany" || method.name == "make") {
                return "<ModelType>[]"
            }
```
`make` method condition should be in the first if... 
Fixed. Next week I'll release 3.2.1 version with this fix. Thank you.

**Comment by gtjamesa** [comment]
Thought it would be something like that! Thanks for fixing so quickly @adelf

**Comment by adelf** [comment]
Fixed in 3.3 version.
